src/Symfony/Component/Form/Resources/translations/validators.fa.xlf

```diff
@@ -4,15 +4,135 @@
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>این مقدار یک رنگ معتبر HTML  نیست.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>لطفاً یک تاریخ تولد معتبر وارد نمایید.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>گزینه‌ی انتخاب‌شده نامعتبر است</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>مجموعه‌ی انتخاب‌شده نامعتبر است.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>لطفاً یک رنگ معتبر انتخاب کنید.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>لطفاً یک کشور معتبر انتخاب کنید.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>لطفاً یک واحد پولی معتبر انتخاب کنید.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>لطفاً یک بازه‌ی زمانی معتبر انتخاب کنید.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>لطفاً یک تاریخ و زمان معتبر وارد کنید.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>لطفاً یک تاریخ معتبر وارد کنید.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>لطفاً یک فایل معتبر انتخاب کنید.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>فیلد مخفی نامعتبر است.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>لطفاً یک عدد صحیح وارد کنید.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>لطفاً یک زبان معتبر انتخاب کنید.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>لطفاً یک جغرافیای (locale) صحیح انتخاب کنید.</target>
```

Comment:
Does it say "**valid** locale" or "**correct** locale"?

It should be "valid".

Reply:
you are correct again! 
you are really good at persian!!!